bluepyemodel/access_point/local.py

```diff
@@ -345,13 +347,10 @@ def store_model_configuration(self, configuration, path=None):
     def get_mechanisms_directory(self):
         """Return the path to the directory containing the mechanisms for the current emodel"""
 
-        if self.emodel_dir:
-            mechanisms_directory = self.emodel_dir / "mechanisms"
-        else:
-            mechanisms_directory = Path("./") / "mechanisms"
+        mechanisms_directory = self.emodel_dir / "mechanisms"
 
         if mechanisms_directory.is_dir():
-            return mechanisms_directory.resolve()
```

Comment:
why remove the .resolve() here?

Reply:
Technically, in the improbable case where the user gives an emodel_dir, the path is not absolute I think. But yes, anyway, it does not harm to keep it ;)

Reply:
You are correct, it might not be absolute in that case, yes let's keep it :)